comms/DiskRepository.cpp

```diff
@@ -291,7 +291,7 @@ class CDiskRepository : public CRepositoryBase
                         if (IsValidExtension(type))
                         {
                             StlLinked<IAttribute> attribute = CreateDiskAttribute(this, moduleName, label, type, pathToWPath(*attr_itr), _T(""), noBroadcast);
-                            if (attribute.isLinked())
+                            if (attribute.isLinked() && boost::iequals(type,_T(".ecl")))
```

Comment:
I think this function is not just used by the autocomplete?  
IOW there are cases where it _should_ return non ecl files?  Specifically the methods takes a "types" param which filters the files already by type- so if the autocomplete code requested files on only type === ECL then all would be good?